src/OrchardCore.Modules/OrchardCore.Email.Azure/Services/AzureEmailSettingsConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using OrchardCore.Email.Azure;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Email.Services
+{
+    public class AzureEmailSettingsConfiguration(IShellConfiguration shellConfiguration, ISiteService site) : IConfigureOptions<AzureEmailSettings>
```

Comment:
I will create another PR for this because when I use `IAsyncConfigureOptions` it throws a runtime except I presume the site settings need to be configured in async fashion

Reply:
The site settings _are_ configured in an async fashion as far as I know. What do you mean, exactly? What's the error?